ci.sh

```diff
@@ -11,7 +11,7 @@ body="{\"authToken\":\"$AUTOMATOR_TOKEN\",\"testSuiteId\":\"$testSuiteId\"}"
 testResults=''
 
 function GetResults () {
-    local bodyPoll="{\"authToken\":\"$AUTOMATOR_TOKEN\",\"buildId\":$1}"
+    local bodyPoll="{\"authToken\":\"$AUTOMATOR_TOKEN\",\"buildId\":\"$1\"}"
```

Comment:
this seems to break some systems, i've removed it until i can understand why some times the buildid ends up as ""BUILDID""

Reply:
the ci below should fail for travis, that runs the Bash script